xarray/backends/api.py

```diff
@@ -480,35 +480,45 @@ def close(self):
 _CONCAT_DIM_DEFAULT = '__infer_concat_dim__'
 
 
-def open_mfdataset(paths, chunks=None, concat_dim=_CONCAT_DIM_DEFAULT,
+def open_mfdataset(paths, chunks=None, concat_dims=_CONCAT_DIM_DEFAULT,
                    compat='no_conflicts', preprocess=None, engine=None,
                    lock=None, data_vars='all', coords='different',
+                   combine='manual',
                    autoclose=None, parallel=False, **kwargs):
     """Open multiple files as a single dataset.
+
+    If combine='auto' then the function `auto_combine` is used to combine the
+    datasets into one before returning the result, and if combine='manual' then
+    `manual_combine` is used. The filepaths must be structured according to
+    which combining function is used, the details of which are given in the
+    documentation for ``auto_combine`` and ``manual_combine``.
+
     Requires dask to be installed. See documentation for details on dask [1].
     Attributes from the first dataset file are used for the combined dataset.
+
     Parameters
     ----------
     paths : str or sequence
         Either a string glob in the form "path/to/my/files/*.nc" or an explicit
-        list of files to open.  Paths can be given as strings or as pathlib
-        Paths.
+        list of files to open. Paths can be given as strings or as pathlib
+        Paths. If concatenation along more than one dimension is desired, then
+        ``paths`` must be a nested list-of-lists (see ``manual_combine`` for
+        details).
     chunks : int or dict, optional
         Dictionary with keys given by dimension names and values given by chunk
         sizes. In general, these should divide the dimensions of each dataset.
         If int, chunk each dimension by ``chunks``.
         By default, chunks will be chosen to load entire input files into
         memory at once. This has a major impact on performance: please see the
         full documentation for more details [2].
-    concat_dim : None, str, DataArray or Index, optional
-        Dimension to concatenate files along. This argument is passed on to
-        :py:func:`xarray.auto_combine` along with the dataset objects. You only
-        need to provide this argument if the dimension along which you want to
-        concatenate is not a dimension in the original datasets, e.g., if you
-        want to stack a collection of 2D arrays along a third dimension.
+    concat_dims : list of str, DataArray, Index or None, optional
```

Comment:
I'm not sure it's worth renaming this argument. We use the singular `dim` for aggregation methods like `mean`, even though you can also pass in a list of dimensions. We'll probably want to still accept scalar strings here, too.

Reply:
Okay noted!